core/webseq-1.0.0/src/webseq.erl

```diff
@@ -79,7 +79,7 @@ stop_pid([Pid, Type]) ->
     catch gproc:unreg(?GPROC_KEY(Type)),
     'ok'.
 
--spec running() -> wh_proplist().
+-spec running() -> [{pid(),_}].
```

Comment:
Should add `-type type_key() :: {?MODULE, ne_binary() | '_' | '$1'}` and use it here and in the `-spec type_key(...)`